source/_components/sensor.glances.markdown

```diff
@@ -44,48 +48,71 @@ sensor:
-      - 'processor_load'
-      - 'process_running'
-      - 'process_total'
-      - 'process_thread'
-      - 'process_sleeping'
-      - 'cpu_temp'
-      - 'docker_active'
-      - 'docker_cpu_use'
-      - 'docker_memory_use'
 ```
 
-Configuration variables:
-
-- **host** (*Required*): The IP address of your host, eg. `192.168.1.32`.
-- **port** (*Option*): The network port to connect to. Default is `61208`.
-- **name** (*Optional*): Name of the Glances sensor.
-- **resources** (*Required*): Entries to monitor.
-  - **disk_use_percent**: Used disk space in percent
-  - **disk_use**: Used disk space
-  - **disk_free**: Free disk space
-  - **memory_use_percent**: Used memory in percent
-  - **memory_use**: Used memory
-  - **memory_free**: Free memory
-  - **swap_use_percent**: Used swap space in percent
-  - **swap_use**: Used swap space
-  - **swap_free**: Free swap space
-  - **processor_load**: Load
-  - **process_running**: Number of running processes
-  - **process_total**: Total number of processes
-  - **process_thread**: Number of threads
-  - **process_sleeping**: Number of sleeping processes
-  - **cpu_temp**: CPU Temperature (may not available on all platforms)
-  - **docker_active**: Count of active Docker containers
-  - **docker_cpu_use**: Total CPU usage in percent of Docker containers
-  - **docker_memory_use**: Total memory used of Docker containers
+{% configuration %}
+host:
+  description: IP address of the host where Glances is running.
+  required: false
+  type: string
+  default: localhost
+port:
+  description: The port where Glances is listening.
+  required: false
+  type: int
+  default: 61208
+name:
+  description: The prefix for the sensors.
+  required: false
+  type: string
+  default: Glances
+version:
+  description: "The version of the Glances API. Supoorted version: `2` and `3`."
+  required: false
+  type: int
+  default: 2
+resources:
+  description: Entries to monitor.
+  required: false
+  type: string
+  default: disk_use
+  keys:
+    disk_use_percent:
+      description: The used disk space in percent.
+    disk_use:
+      description: The used disk space.
+    disk_free:
+      description: The free disk space.
+    memory_use_percent:
+      description: The used memory in percent.
+    memory_use:
+      description: The used memory.
+    memory_free:
+      description: The free memory.
+    swap_use_percent:
+      description: The used swap space in percent.
+    swap_use:
+      description: The used swap space.
+    swap_free:
+      description: The free swap space.
+    processor_load:
+      description: The load.
+    process_running:
+      description: The number of running processes.
+    process_total:
+      description: The total number of processes.
+    process_thread:
+      description: The number of threads.
+    process_sleeping:
+      description: The number of sleeping processes.
+    cpu_temp:
+      description: The CPU temperature (may not available on all platforms).
```

Comment:
"may not be available" or "not available"

Reply:
Thanks